upgrade/v1beta1/deploy/install/01-prerequisite/02-serviceaccount.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
```

Comment:
TBH, I don't understand _why_ we need to have these files
```
v1beta1/deploy/*
```

Reply:
I think we don't need, actually. Just copy the files like the v1alpha1.